app/src/pages/admin/allUsersPage/usersToolbar/actionPanel/actionPanel.jsx

```diff
@@ -49,6 +49,10 @@ const messages = defineMessages({
     id: 'ActionPanel.addUserNotification',
     defaultMessage: 'New account has been created successfully',
   },
+  memberWasInvited: {
+    id: 'MembersPage.memberWasInvited',
+    defaultMessage: "Member '<b>{name}</b>' has assigned to the project",
```

Comment:
Should also be added if we invite user by email as in https://jiraeu.epam.com/browse/EPMRPP-38474.

Reply:
It's already works properly.